### PR TITLE
using regex to remove ansi escape for colors

### PR DESCRIPTION
**Pull request by ewmassey**
This removes the color escape chars so that it doesn't muddy up the exception outputs.

### CODE REVIEW

**Comment by rgbkrk** [comment]
This is only for while papermill is running, right? Not for inside the notebook where we can have all the ansi codes we like?

**Comment by adparker** [comment]
Thanks for the quick turn around!

File: papermill/exceptions.py [hunk 1]
```diff
@@ -21,8 +23,11 @@ def __init__(self, exec_count, source, ename, evalue, traceback):
         self.traceback = traceback
         message = "\n" + 75 * "-" + "\n"
         message += 'Exception encountered at "In [%s]":\n' % str(exec_count)
+        tb = ""
         for line in traceback:
-            message += line + "\n"
+            tb += line + "\n"
+        ansi_escape = re.compile(r'\x1b[^m]*m')
```
Comment (rgbkrk) [inline papermill/exceptions.py]: The complete regex you probably want is `r'\x1b\[[\d;]+m`. However, there is a module called `ansiwrap` with a function `strip_color`.

**Comment by rgbkrk** [comment]
:shipit:

**Pull request merged**